bokeh.oggm.org/docker-compose.yml

```diff
@@ -83,3 +83,18 @@ services:
       - "traefik.http.services.hma_glacial_lakes-app.loadbalancer.server.port=8080"
       - "traefik.http.routers.hma_glacial_lakes-app.entrypoints=web"
       - "traefik.http.routers.hma_glacial_lakes-app.rule=PathPrefix(`/hma_glacial_lakes`)"
+
+  dtcg_l2_dashboard-app:
+    image: "ghcr.io/oggm/bokeh:20230101"
+    container_name: "dtcg_l2_dashboard-app"
+    command:
+      - "git+https://github.com/DTC-Glaciers/dtcg-web.git@v0.1.1"
+      - "fastapi run dtcgweb/main.py"
```

Comment:
I don't think this works? Since this will be passed as the "app" parameter to "panel serve".